tests/test_apply.py

```diff
@@ -48,9 +48,7 @@
     ("PauliZ", Z),
     ("Hadamard", H),
     ("S", S),
-    ("Sdg", S.conj().T),
     ("T", T),
```

Comment:
You could add the following here,
```suggestion
    ("T", T),
    ("S.inv", S.conj().T),
    ("T.inv", T.conj().T),
```
and this should work right? Might be good to leave here as a uni ttest.

Reply:
I believe these were included since there were separate ``Sdg`` and ``Tdg`` gates defined, now I'd say the unit tests in ``test_inverses.py`` should suffice. What do you think?

Reply:
The tests in `test_inverses.py` are more like integration tests, however, in that they make use of the PennyLane QNode and PennyLane operation. This test is much more of a unit test, avoiding most external code and simply calling `device.apply()` directly.

I think it's good to have both, as they serve different purposes;

* If something breaks in `test_inverses.py` but not `test_apply`, we know the issue is with the PL-Core/PL-qiskit integration.

* If something breaks in `test_apply` _and_ `test_inverses.py`, we know that the issue is likely with the plugins' use of the Qiskit API.